src/NeXus.def

```diff
@@ -0,0 +1,84 @@
+LIBRARY NeXus.dll
```

Comment:
@FreddieAkeroyd  Just for my own curiosity: does this replace the `dllexport` and `dllimport` in the code? This would be good to know also for my own projects ;)